src/utils.ts

```diff
@@ -182,8 +182,8 @@ export const getActiveSectionsWithData = (
 
     if (Array.isArray(sectionData)) {
       const section = {
-        ...sectionConfig,
```

Comment:
@Mudaafi let me know if I'm thinking about this incorrectly but I think order of the spread (...) vs `data` is important here. For custom sections, we want new `sectionData` to override the one coming from the config.
<img width="670" alt="Screenshot 2024-01-27 at 1 34 45 PM" src="https://github.com/Constructor-io/constructorio-ui-autocomplete/assets/2954126/2b35665c-14ca-4f1c-9217-ae04cdfc1850">
<img width="617" alt="Screenshot 2024-01-27 at 1 33 49 PM" src="https://github.com/Constructor-io/constructorio-ui-autocomplete/assets/2954126/52e20d7b-586d-4f8e-8899-c5042948e0e0">

Reply:
Should sectionConfig have data for other sections than custom? I think it shouldn't 🤔

Reply:
Correct, only for `custom`

Reply:
So why would we worry if sectionConfig overrides sectionData then?

Reply:
Because we don't want that for custom sections and this code runs for custom sections? The screenshot above is from a custom section. Am I understanding this wrong?

Reply:
What I mean is it's ok for custom sections if it overrides the data since we're not handling it anyway

Reply:
On line 173 we are modifying the data being passed to us then here we are overriding it back

Reply:
Aha got it

Reply:
Just so I'm on the same page, are we saying the order of priority is:
1. Dashboard/Catalog set configs
2. Developer-set top-level-component configs

That didn't feel intuitive to me :thinking: but maybe it's because I'm thinking of it as an "override" rather than a "fallback"

Reply:
Okay nvm scratch that, I get the issue with `data` getting overriden